packages/tailwindcss/src/utilities.ts

```diff
@@ -310,12 +310,15 @@ export function createUtilities(theme: Theme) {
       let value: string | null = null
 
       if (!candidate.value) {
+        if (candidate.modifier) return
```

Comment:
I feel like this is wrong but I can't remember which "default" utility was/is supposed to take a modifier (this was a bug in v3 iirc).

But if we can't come up with any then maybe it's fine.